services/visual-studio-marketplace/visual-studio-marketplace-last-updated.tester.js

```diff
@@ -0,0 +1,79 @@
+'use strict'
+
+const t = (module.exports = require('../tester').createServiceTester())
+const { isFormattedDate } = require('../test-validators')
+
+t.create('date')
+  .get('/visual-studio-marketplace/last-updated/yasht.terminal-all-in-one.json')
+  .expectBadge({
+    label: 'last updated',
+    message: isFormattedDate,
+  })
+
+t.create('date')
+  .get('/visual-studio-marketplace/last-updated/yasht.terminal-all-in-one.json')
+  .intercept(nock =>
+    nock('https://marketplace.visualstudio.com/_apis/public/gallery/')
+      .post(`/extensionquery/`)
+      .reply(200, {
+        results: [
+          {
+            extensions: [
+              {
+                statistics: [
+                  {
+                    statisticName: 'averagerating',
+                    value: 2.5,
+                  },
+                  {
+                    statisticName: 'ratingcount',
+                    value: 10,
+                  },
+                ],
+                versions: [
+                  {
+                    version: '1.0.0',
+                  },
+                ],
+                releaseDate: '2013-04-13T07:50:27.000Z',
+                lastUpdated: '2013-04-13T07:50:27.000Z',
+              },
+            ],
+          },
+        ],
+      })
+  )
+  .expectBadge({
+    label: 'last updated',
+    message: 'april 2013',
+    color: 'red',
+  })
```

Comment:
This looks like a mocked version of the live test above, so let's remove it. Our service tests are largely a suite of integration tests with the myriad upstream services and platforms Shields integrates with, so we try to be mindful about having duplicative tests within here. 

I know some of the other VS Marketplace service tests have similar ones, but we don't need to mimic that here.

Reply:
Got it! It's removed.